src/assets/sass/protocol/components/forms/_form.scss

```diff
@@ -15,11 +15,12 @@
 
 
 .mzp-c-form {
-    color: $form-text;
+    color: get-theme('body-text-color-alt');
```

Comment:
I would prefer to keep using the `$form-text` variable since that has now been updated to use the theme mixin.